nerfstudio/configs/base_config.py

```diff
@@ -148,3 +151,9 @@ class ViewerConfig(PrintableConfig):
     actually used in training/evaluation. If -1, display all."""
     quit_on_train_completion: bool = False
     """Whether to kill the training job when it has completed. Note this will stop rendering in the viewer."""
+    image_format: Literal["jpeg", "png"] = "png"
```

Comment:
I think the default should stay jpeg, but we can bump the quality to 85 or 90

Reply:
For remote usage, I can see using JPEG by default, but would be good to increase quality.

For local usage where not bandwidth constrained, PNG looks much nicer IMO.

I wonder whether the default should be "auto" or something.